### ISSUE TITLE
Error messages are not announced for screen readers

### ISSUE THREAD

**Issue by dominoweir** [t49tran/react-google-recaptcha-v3#203]
The status of the checkbox for recaptcha is announced correctly with a screen reader turned on but the expiration message is not. This is confusing for screen reader users, and could probably fixed by turning the DOM element for the error message into a live region ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)). 

To reproduce: 

1. Turn on your device's screen reader (Cmd+F5 on my Mac to turn on VoiceOver)
2. Load any page or component that contains a recaptcha
3. Check the recaptcha text box and note that it is announced as checked by the screen reader
4. Wait one minute for the recaptcha to expire until the error message pops up
5. Note that the error message is not announced

**Comment by wojtekmaj** [comment]
This is outside of scope of this package. The checkbox and everything around it is actually created by Google's reCAPTCHA script. This package just allows it to load in a react-y way. Unfortunately you'd need to have a chat with Google about it.
